wandb/sdk_py27/internal/sender.py

```diff
@@ -227,6 +234,20 @@ def send_request_stop_status(self, record):
                 logger.warning("Failed to check stop requested status: %s", e)
         self._result_q.put(result)
 
+    def debounce(self):
+        config_value_dict = self._config_format(self._consolidated_config)
```

Comment:
why not just:
```

if self._debounced_config_value_dict is not None:
   do upsert
   self._config_save(self._debounced_config_value_dict)
self._debounced_config_value_dict = None
```

Reply:
did an even simpler version with just a boolean.